### ISSUE TITLE
[Bug] UWP Focus Regression from PR #11140

### ISSUE THREAD

**Issue by PureWeen** [xamarin/Xamarin.Forms#11348]
### Description

https://github.com/xamarin/Xamarin.Forms/commit/25ffd04152f2e29fd740fe94d0cfc05a2c16482d

Is now causing the `Focus View` UI test on UWP to fail

### Steps to Reproduce

1. Start Control Gallery
2. Go to Entry Gallery
3. Select `Focus View`

At this point you should se

Focus? False

Or run the UI Test for `EntryUITests => _Focus`

### Expected Behavior
When you navigate to a page the Entry shouldn't be the first thing focused by default (that's at least my understanding of this UI Test)

### Actual Behavior
The Entry is the first thing that has focus

### Screenshots

![image](https://user-images.githubusercontent.com/5375137/86854590-4c7e9d80-c07e-11ea-97c1-d5fa9945e84d.png)

**Comment by PureWeen** [comment]
FYI @bmacombe

You can see the failure here on the CI
https://dev.azure.com/xamarin/public/_releaseProgress?_a=release-environment-extension&releaseId=1927&environmentId=72181&extensionId=ms.vss-test-web.test-result-in-release-environment-editor-tab&runId=1105740&resultId=100011&paneView=debug

**Comment by bmacombe** [comment]
@PureWeen Thanks...I'll check it out, I thought I looked at that page in the gallery, but I must have misread the before result.

**Comment by bmacombe** [comment]
The Focus() call was needed to fix #8787...#2171, #8503 are fine without it.  So I'll see if I can find another method to fix 8787

**Comment by PureWeen** [comment]
awesome thank you @bmacombe

**Comment by bmacombe** [comment]
@PureWeen I think I found another solution, but I'm having issues with the UI tests.  I believe I had this before.

I've tried both the 1.1 and 1.2 WinAppDriver but the automation has issues bring up the various test pages.  It misses the first level  Any ideas?

![image](https://user-images.githubusercontent.com/15127788/86862672-4644ed00-c08f-11ea-836c-7248d762edea.png)

I think the issue is with the Packager being called too soon, I investigated this when I was working on #2172.  It's a pretty low-level change so I want to make sure it doesn't break anything else.

![image](https://user-images.githubusercontent.com/15127788/86862794-899f5b80-c08f-11ea-8b94-67aaccb8d5dd.png)

**Comment by bmacombe** [comment]
@PureWeen PR #11350 will remove that focus call and should at least unblock this issue.  I'll split a fix for #8787 into another PR since it might take more review

**Comment by bmacombe** [comment]
#11351 is part two of the fix and should be reviewed well before merging.

**Comment by PureWeen** [comment]
@bmacombe do you think any of the changes regressed the following tests?

Issue852TestsEntriesClickable
https://github.com/xamarin/Xamarin.Forms/issues/11352
TestShowContextMenuItemsInTheRightOrder

**Comment by bmacombe** [comment]
@PureWeen I don't think so, I'm not sure how any of the changes in 11140 would affect the grid.  I ran the test locally and it passed at the 4.7 head.  I also tried at the last commit before 11140 was merged and it passed.

![image](https://user-images.githubusercontent.com/15127788/86874814-c460be00-c0a6-11ea-9f7e-96bddca9863c.png)

**Comment by bmacombe** [comment]
@PureWeen 852 might be related since the Focus goes to the last Entry, which it shouldn't

**Comment by bmacombe** [comment]
@PureWeen I don't think so on 2414 TestShowContextMenuItemsInTheRightOrder.

Tested 4.7 head and 11350 both passed on UWP

**Comment by samhouts** [comment]
closed by #11350
